storage/innobase/fts/fts0fts.cc

```diff
@@ -4375,27 +4372,29 @@ dberr_t fts_sync_table(dict_table_t* table, bool wait)
 	return(err);
 }
 
-/** Check if a fts token is a stopword or less than fts_min_token_size
-or greater than fts_max_token_size.
+/** Check if a fts token is a stopword.
 @param[in]	token		token string
 @param[in]	stopwords	stopwords rb tree
-@param[in]	cs		token charset
-@retval	true	if it is not stopword and length in range
-@retval	false	if it is stopword or lenght not in range */
+@retval	true	if it is a stopword
+@retval	false	if it is not a stopword */
 bool
-fts_check_token(
+fts_token_is_stopword(
 	const fts_string_t*		token,
-	const ib_rbt_t*			stopwords,
-	const CHARSET_INFO*		cs)
+	const ib_rbt_t*			stopwords)
 {
```

Comment:
How does this change affect the file format compatibility with existing InnoDB tables that contain `FULLTEXT INDEX` and were created or altered in MySQL 5.6, 5.7, or MariaDB 10.0 or later?

Reply:
These changes should have no effect on file format at all, since they only move where the limits on word lengths are checked. The changes also maintain default fulltext engine behavior, and only change how limits are enforced on pluggable fulltext parsers. If somebody used their own fulltext parser plugins, their behavior may change: more words will get their way into indexes.

This particular function `fts_check_token()` didn't use its parameter `cs`, so its removal will have no effect.